_episodes/11-dask-configuration.md

```diff
@@ -0,0 +1,390 @@
+---
+title: "Configuring Dask"
+teaching: 20 (+ optional 10)
+exercises: 40 (+ optional 20)
+compatibility: ESMValCore v2.10.0
+
+questions:
+  - "What is the Dask configuration file and how should I use it?"
+  - "What are Dask workers"
+  - "What is the Dask scheduler"
+
+objectives:
+  - "Understand the contents of the dask.yml file"
+  - "Prepare a personalized dask.yml file"
+
+keypoints:
+  - "The ``~/.esmvaltool/dask.yml`` file tells ESMValCore how to configure Dask."
+  - "``cluster`` can be used to start a new Dask cluster for each run."
+  - "``client`` can be used to connect to an already running Dask cluster."
+  - "The Dask default scheduler can be configured by editing the files in ``~/.config/dask``."
+  - "The Dask Dashboard can be used to see if the Dask workers have sufficient memory available."
+
+---
+
+## Introduction
+
+When processing larger amounts of data, and especially when the tool crashes
+when running a recipe because there is not enough memory available, it is
+usually beneficial to change the default
+[Dask configuration](https://docs.esmvaltool.org/
```

Comment:
Suggest adding longer web url links to links.md file.

Reply:
Done